src/ceph-daemon/ceph-daemon

```diff
@@ -1623,8 +1623,9 @@ def _get_parser():
         formatter_class=argparse.ArgumentDefaultsHelpFormatter)
     parser.add_argument(
         '--image',
-        default=DEFAULT_IMAGE,
-        help='container image')
+        default=os.environ.get('CEPH_DAEMON_IMAGE', DEFAULT_IMAGE),
```

Comment:
> To me it would feel more intuitive if the precedence was defined as:
> `CLI_ARG > ENV > CONFIG_STORE > DEFAULT`

Actually, looks like the command line arg is already preferred over the env ... ?